### PR TITLE
Feature branch/rails forms

### PR DESCRIPTION
**Pull request by bini-i**
## This is a rails form project from microverse rails curriculum with features of 
- Creating, 
- Reading, 
- Updating, and 
- Deleting a user.

### CODE REVIEW

File: app/controllers/users_controller.rb
Comment (voscarmv) [inline app/controllers/users_controller.rb]: Please make sure to `redirect_to` **only** if `@user.save` was successful. Otherwise, just `render :edit`. You may use an `if` statement for this.

File: app/views/users/new.html.erb
Comment (voscarmv) [inline app/views/users/new.html.erb]: Your commented form here should use the `users_path` helper function, thus:

```
`<form action="users_path" method="post" accept-charset="UTF-8">`
```

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team 💻 👀 ✨ 

- Most of the project requirements have been met.

## However, there are some changes required 👀 🧰 🔨 🔧 

- Please only use the `:new, :create, :edit, :update` routes as specified by Odin: 
image
- Please check my code comments for details on fixing your project!


### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

File: app/controllers/users_controller.rb
Comment (teekaytech) [inline app/controllers/users_controller.rb]: - [ ] `redirect_to new_user_path` if user was saved ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

File: app/controllers/users_controller.rb
Comment (teekaytech) [inline app/controllers/users_controller.rb]: - [ ] Use `redirect_to edit_user_path` if user was saved

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this project. You've also done a great job in fixing the required changes by the previous reviewer.
However, some changes are still required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] I added some inline comments to your code. Kindly check and fix them as appropriate.

- [x] As mentioned by the previous reviewer, your commented form here should use the users_path helper function, thus:

`<form action="users_path" method="post" accept-charset="UTF-8">`



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: app/views/users/new.html.erb
Comment (peterrobert) [inline app/views/users/new.html.erb]: This field should not be a text_field. It should be an email_field. This is important because it makes sure that what is added in the field is only an email address.

File: app/views/users/edit.html.erb
Comment (peterrobert) [inline app/views/users/edit.html.erb]: Kindly make sure that this is an email_field

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :100: 
HELLO @bini-i :wave: VERY NICE PROJECT. I LOVE THAT YOU HAVE WORKED ON ALL THE CHANGES SUGGESTED BY THE PREVIOUS TSE. HOWEVER, THERE IS ONE MORE CHANGE THAT IS REQUIRED BEFORE YOU CAN MOVE ON KINDLY CHECK BELOW :arrow_down: 

### [OPTIONAL]

- [ ] You can create a partial for the user form and use it for both the edit and the new HTML views.

if you would like to learn more about partials kindly check [here](https://guides.rubyonrails.org/layouts_and_rendering.html);


### if you have any questions kindly reach me on slack @peterrobert

### HAPPY CODING :computer:

**Review by peterrobert — APPROVED** [review]
### ###  APPROVED :+1: :100: 

VERY NICE WORK. GOOD JOB IMPLEMENTING ALL THE REQUIRED CHANGES.

ON TO THE NEXT PROJECT.

### HAPPY CODING :computer:

**Pull request merged**
